geoips/plugins/modules/output_checkers/image.py

```diff
@@ -131,19 +132,14 @@ def outputs_match(plugin, output_product, compare_product, threshold=0.05):
     # If shapes of arrays do not match, pixel diff can not be performed.
     # Print the names of the two images and associated shapes, and return False.
     else:
-        LOG.interactive("    ***************************************")
-        LOG.interactive("    *** BAD Images NOT match exactly, different sizes ***")
-        LOG.interactive(
-            "    ***   output_product: %s %s ***",
-            np.array(out_img).shape,
-            output_product,
+        log_with_emphasis(
+            LOG.interactive,
+            ["BAD Images NOT match exactly, different sizes"],
         )
-        LOG.interactive(
-            "    ***   compare_product: %s %s ***",
-            np.array(comp_img).shape,
-            compare_product,
-        )
-        LOG.interactive("    ***************************************")
+        message = f"output_product: {np.array(out_img).shape} {output_product}"
+        log_with_emphasis(LOG.interactive, [message])
+        message = f"compare_product: {np.array(comp_img).shape} {compare_product}"
+        log_with_emphasis(LOG.interactive, [message])
```

Comment:
This should just be one log_with_emphasis call, with all of the lines in a single messages list?

Reply:
Yes, this should be multiple lines passed into a single log_with_emphasis call, please.

Reply:
Fixed